HW3_new/src/routes/list.tsx

```diff
@@ -0,0 +1,47 @@
+import Root from './root';
+import React, { useState, useEffect } from 'react';
```

Comment:
Country list shows nicely but each card has lot of space on the right side the content of the card could be centered for nicer view.

Reply:
Content is now centered